webrtc.html

```diff
@@ -2716,8 +2716,8 @@
                         <p>In parallel, add the ICE candidate
                         <var>candidate</var> as described in <span data-jsep=
                         "addicecandidate">[[!JSEP]]</span>. Use
-                        <code><var>candidate</var>.ufrag</code> to identify the
-                        ICE <a>generation</a>; if the ufrag is null, process the
+                        <code><var>candidate</var>.usernameFragment</code> to identify the
+                        ICE <a>generation</a>; if the usernameFragment is null, process the
```

Comment:
Here we're talking about the member, not the username fragment in the abstract sense, so should be `<code>usernameFragment</code>`.